README.md

```diff
@@ -1 +1,73 @@
+Extension Manifest Converter is an open source tool to convert existing Chrome extensions to
+Manifest V3. Use it to convert an entire directory, extension zip file, or just a manifest.json
+file. All expected changes are applied to manifest.json.
+
+## Features
+
+* Performs conversions on
+  * unpacked extension directories
+  * zip files containing an extension
+  * manifest.json
+* General manifest.json conversions
+  * Updates `manifest_version` field
+  * Converts between host permissions declared in `permissions` or `optional_permissions` in MV2 and
+    `host_permissions` in MV3
+  * Converts between a `content_security_policy` string in MV2 and `content_security_policy` object
+    with`extension_pages` and `sandbox` properties in MV3
+  * Converts between `background.scripts` in MV2 and background service workers
+    `background.service_worker` in MV3
+* Scripting API conversions
+  * Converts `chrome.tabs.executeScript` in MV2 to `chrome.scripting.executeScript` in MV3. If
+    necessary, also adds `scripting` to the `permissions` array in manifest.json.
+  * Converts `chrome.tabs.insertCSS` in Mv2 to `chrome.scripting.insertCSS` in MV3. If necessary,
+    also adds `scripting` to the `permissions` array in manifest.json.
+* Action API conversions
+  * Converts calls to `chrome.browserAction` and `chrome.pageAction` in MV2 into `chrome.action` in
+    MV3
+  * Converts `browser_action` and `page_action` manifest entries in MV2 into `action` in MV3
+
+## Limitations
+
+This tool aims to simplify the MV3 conversion; it does not fully automate the process. Only search
+and replace changes are applied to `.js` files.
+
+This tool does not:
+
+* update any service worker code that relies on a DOM
```

Comment:
Perhaps a hint that if their BG script relies on the DOM, they'll need to spin off part as a content script?

Reply:
It's not quite that easy. There are a ton of different ways an author might be using DOM and only some of them make sense in a content script. I'd rather not get into migration guidance here.